app/src/main/java/tech/ula/ui/FilesystemEditFragment.kt

```diff
@@ -233,7 +233,8 @@ class FilesystemEditFragment : Fragment() {
             navController.popBackStack()
         } else {
             try {
-                filesystem.archType = DeviceArchitecture().getArchType()
+                val ulaFiles = UlaFiles(activityContext, activityContext.applicationInfo.nativeLibraryDir)
```

Comment:
It doesn't look like this needs to be surrounded in a `try catch` block anymore.

Reply:
I removed the `try/catch`